src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -291,6 +291,7 @@
     <Compile Include="..\NuGet.Packaging\PackagingConstants.cs" Link="NuGet.Packaging\PackagingConstants.cs" />
     <Compile Include="..\NuGet.Packaging\VersionFolderPathResolver.cs" Link="NuGet.Packaging\VersionFolderPathResolver.cs" />
     <Compile Include="..\NuGet.Packaging\Sha512HashFunction.cs" Link="NuGet.Packaging\Sha512HashFunction.cs" />
+    <Compile Include="..\NuGet.Packaging\StringFormatter.cs" Link="SNuGetPackaging\StringFormatter.cs" />
```

Comment:
what's the S here for?

Reply:
typo, although it's hard to guess what I was going for. 😕